internal/ingest/linksystem.go

```diff
@@ -324,39 +352,35 @@ func (ing *Ingester) syncAdEntries(from peer.ID, ad schema.Advertisement, adCid,
 			// Do not mark ad as done so that it will be reprocessed.
 			return
 		}
-		// Processed all the entries, so mark ths ad as processed.
+		// Processed all the entries, so mark this ad as processed.
 		if err := ing.markAdProcessed(from, adCid); err != nil {
 			log.Errorw("Failed to mark ad as processed", "err", err)
 		}
 	}()
 
-	isRm, err := ad.FieldIsRm().AsBool()
-	if err != nil {
-		log.Errorw("Cannot read IsRm field", "err", err)
+	// This ad has bad data or has all of its content deleted by a subsequent
+	// ad, so skip any further processing.
+	if skip {
 		return
```

Comment:
Log a message for debugging purposes?

Reply:
A message is logged whereever skip is set to true.